kuzzleio/sdk-java
# Add rules to generate and publish the .jar

## What does this PR do ?

:warning: Depends on #46 :warning: 

Add rules to generate and publish the .jar

## How to test

Run
```sh
./gradlew jar
```

There should be `sdk-java-3.0.0.jar` under `build/libs`

## Other changes

Added gradle wrapper
